pkg/odo/genericclioptions/clientset/clientset.go

```diff
@@ -236,14 +238,20 @@ func Fetch(command *cobra.Command, platform string) (*Clientset, error) {
 		dep.BindingClient = binding.NewBindingClient(dep.ProjectClient, dep.KubernetesClient)
 	}
 	if isDefined(command, PORT_FORWARD) {
-		dep.PortForwardClient = portForward.NewPFClient(dep.KubernetesClient, dep.StateClient)
+		switch platform {
+		case commonflags.PlatformPodman:
+			dep.PortForwardClient = podmanportforward.NewPFClient(dep.ExecClient)
```

Comment:
I think you will need to add EXEC as dependency of PORT_FORWARD in `subdeps`.

I works as is now because EXEC is added as a direct dependency of the dev command, but this could change in the future

Reply:
Good point indeed! I've updated the sub-deps in [`7b384d3` (#6629)](https://github.com/redhat-developer/odo/pull/6629/commits/7b384d3aa85641ed81a15615ceea713df126ea04). Also noted that `EXEC` might also be missing the `PODMAN_NULLABLE`.